### PR TITLE
Resolved TROCR Microsoft Large Printed Bug #14238

### PR DESCRIPTION
**Pull request by khanfarhan10**
Original Image Output Size : (X,Y)
Original Std Dev and Mean Shape : (3,)

Reshaping the Image to Shape : (1,X,Y) solves the bug, which is implemented in the code!

# What does this PR do?

Fixes #14239 

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by khanfarhan10** [comment]
Hmm. That's strange, check_code_quality fails.

Can the mods guide me n how to improve that? CircleCI says :

```
#!/bin/bash -eo pipefail
black --check examples tests src utils
would reformat src/transformers/image_utils.py
Oh no! 💥 💔 💥
1 file would be reformatted, 1139 files would be left unchanged.

Exited with code exit status 1
CircleCI received exit code 1
```

**Comment by LysandreJik** [comment]
Hello! You can fix the code quality issue by running the following commands at the root of your clone:

```
pip install -e .[quality]
make fixup
```

This should fix every issue that can be fixed and let you know if some issues need manual intervention. Thank you!

**Comment by LysandreJik** [comment]
cc @NielsRogge can you take a look at this and review? If you approve, I'm happy to push the style fixes on your branch @khanfarhan10

File: src/transformers/image_utils.py [hunk 1]
```diff
@@ -142,7 +142,11 @@ def normalize(self, image, mean, std):
                 mean = torch.tensor(mean)
             if not isinstance(std, torch.Tensor):
                 std = torch.tensor(std)
+                
+        if len(image.shape) == 2 :
+            image = image.reshape(1, image.shape[0],image.shape[1])
 
+
```
Comment (NielsRogge) [inline src/transformers/image_utils.py:149]:
```suggestion
```

File: src/transformers/image_utils.py [hunk 2]
```diff
@@ -142,7 +142,11 @@ def normalize(self, image, mean, std):
                 mean = torch.tensor(mean)
             if not isinstance(std, torch.Tensor):
                 std = torch.tensor(std)
+                
+        if len(image.shape) == 2 :
+            image = image.reshape(1, image.shape[0],image.shape[1])
```
[the remark this answers is not in the document]
Comment (sgugger) [reply src/transformers/image_utils.py:147]: Indeed! Normalization with three channels does not make sense for images with just two dimensions, so a warning here to explain we added one would be great (users should have added that in their own preprocessing).

**Comment by khanfarhan10** [comment]
Apologies for not being active on github for quite sometime guys, seem to have completely missed this thread!

Great to see it looks [already handled](https://github.com/huggingface/transformers/blob/3a8eb74668e9c2cc563b2f5c62fac174797063e0/src/transformers/image_utils.py#L613), thanks for the effort and a yet amazing library!
